spec/features/search_by_food_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe "Food Search" do
+  it "Can search for foods by ingredient" do
+    visit root_path
+    fill_in :q, with: "sweet potatoes"
+    click_on "Search"
+    expect(current_path).to eq foods_path
+    expect(page).to have_content("Foods Found: 10")
+    expect(page).to have_css('.foods', count: 10)
+    within(first('.foods')) do
+      expect(page).to have_content("GTIN/UPC code: 492111402857")
+      expect(page).to have_content("Description: SWEET POTATOES")
+      expect(page).to have_content("Brand Owner: ARCHER FARMS")
+      expect(page).to have_content("Ingredients: SWEET POTATOES")
+    end
```

Comment:
Nice job with testing the elements on the page. You might also consider making your class names a bit more dynamic, for example, you could interpolate the food id into the class name, that way you could target specific elements of the page and test what's within each section for a specific dish.